layouts/index.html

```diff
@@ -30,12 +30,7 @@
 
 {{ define "main" }}
   {{ partial "info/basic.html" . }}
-  {{ partial "info/challenges.html" . }}
   {{ partial "info/features.html" . }}
-  {{ partial "info/schedule.html" . }}
-  {{ partial "info/venue.html" . }}
-  {{ partial "info/register.html" . }}
   {{ partial "info/team.html" . }}
-  {{ partial "info/partners.html" . }}
```

Comment:
is it required to remove them from the layout? Ideally these should be hidden if they are not defined in the config file instead, so that the theme can be used in any point in time during planning.

Reply:
I tried to comment them out like this:

`{{ /* partial "info/schedule.html" . */ }}`

but it didn't work.. Hiding via a css class would be fine too. This was quickest and dirtiest ;-)

Reply:
I'm not sure how you would configure them to be hidden.